app/javascript/reducer/reducer.js

```diff
@@ -0,0 +1,28 @@
+import Axios from "axios";
+
+const ADDGREETING = "ADDGREETING";
+
+export const addGreeting = (greeting) => ({ type: ADDGREETING, payload: greeting });
+
+export const fetchGreeting = () => async (dispatch) => {
+    try{
+        const res = await Axios.get('http://127.0.0.1:3000/api/v1/greetings');
```

Comment:
- **[OPTIONAL]** Since the FE and API are on the same location, to avoid CORS policy blocking requests you don't need to specify this entire URL. You can simply change to this one:

  ```
  const res = await Axios.get('/api/v1/greetings');
  ```

Reply:
thanks